src/components/core/PivotTable.tsx

```diff
@@ -582,38 +618,43 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
             (!alreadyResized() || (alreadyResized() && force))
         ) {
             this.resizing = true;
+            // we need to know autosize width for each column, even manually resized ones
```

Comment:
Maybe we could add one general comment at the beginning which would describe this even more, focusing on the order of events (auto, growtofit, manual) and their relations

Reply:
logic moved to external function + UT + comment